pyface/ui/qt4/tasks/tests/test_split_editor_area_pane.py

```diff
@@ -382,6 +382,35 @@ def test_context_menu_merge_text_top_bottom_split(self):
         with event_loop():
             window.close()
 
+    def test_no_context_menu_if_outside_tabwidgets(self):
+        # Check that the case of a position not in any of the tabwidgets
+        # is handled correctly.
+        window = TaskWindow()
+
+        task = SplitEditorAreaPaneTestTask()
+        window.add_task(task)
+
+        with event_loop():
+            window.open()
+
+        editor_area = task.editor_area
+        editor_area_widget = editor_area.control
+        tab_widget, = editor_area_widget.tabwidgets()
+
+        context_menu_event = QtGui.QContextMenuEvent(
+            QtGui.QContextMenuEvent.Mouse,
+            # Position relative to the widget, so (-1, -1) should be
+            # reliably outside.
+            QtCore.QPoint(-1, -1),
+        )
+
+        # Just exercise the context menu code to make sure it doesn't raise.
```

Comment:
This could also do with a test to check directly that `get_context_menu` is returning `None`. I'll add that.

Reply:
Done.